jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -141,15 +189,15 @@ public void setup() {
         vantiq = new io.vantiq.client.Vantiq(testVantiqServer);
         vantiq.setAccessToken(testAuthToken);
     }
-    
-    /* Uncomment this BeforeClass when debugging to ensure tables have been properly deleted.
-    
+
     @BeforeClass
-    public static void debugSetup() throws VantiqSQLException {
-        tearDown();
+    public static void setupEnv() throws IOException {
```

Comment:
Handle failing tests that hadn't been changed after config changes.  Problem with too many setup variables needed....